hack/codegen-naming-exceptions.yaml

```diff
@@ -0,0 +1,8 @@
+plural:
+  # These exceptions are used to avoid bad pluralization (Endpoints->Endpointses, etc..)
+  Endpoints: Endpoints
+public:
+  # These exceptions are used to deconflict the generated code. You can put your
+  # fully qualified package to generate a name that does not conflict with your
+  # group.
```

Comment:
Not sure I understand this from the comment. What about private below?

Reply:
I tried to copy what we had before in code, not sure if I understand it properly either... The private has the same comment, basically just decapitalize the value. I'm open for the comment suggestions...